src/main/java/seedu/address/MainApp.java

```diff
@@ -206,6 +207,28 @@ protected void initImportsDirectory() {
         }
     }
 
+    /**
+     * Sets up the data with the given password.
+     *
+     * @param input The input password from user.
+     * @return {@code true} if the password is correct, {@code false} otherwise
```

Comment:
This only returns true

Reply:
Oh no, I shouldn't have merged the PR. @Zhou-Jiahao-1998 if this method only returns `true`, it makes sense for this method to be of `void` return type instead.

Reply:
@Zhou-Jiahao-1998 just continue your fix and push to this branch, then create a new pull request again.